src/universal/decorators/socketWithPresence/socketWithPresence.js

```diff
@@ -31,9 +31,9 @@ const mapStateToProps = (state, props) => {
 };
 
 export default ComposedComponent => {
+  @requireAuth
```

Comment:
requireAuth should always be the first thing we check. We shouldn't attempt to open a websocket before we guarantee that they are logged in.